config/mainnet/constants.ts

```diff
@@ -1593,11 +1601,13 @@ const constants: IConstants = {
         ...BLACKLIST.BY_ACCOUNTS_THIEF,
     ]),
     THIEVES_ADDRESSES_BLACKLIST: BLACKLIST.TRANSACTIONS_IN_THE_PAST,
+    FRAUD_OPERATION_BLACKLIST: BLACKLIST.FRAUD_OPERATION,
     START_FEATURE_BLOCK: {
         ACCOUNT_BLACKLIST: 70000,
         THIEVES_ACCOUNT_BLACKLIST: 455255,
         VOTE_WITH_ACTIVE_STAKE: 245000,
         FIX_TRANSACTION_IN_THE_PAST_BLOCK_HEIGHT: 455255,
+        FRAUD_OPERATION_BLACKLIST: 697842,
```

Comment:
is this height of block?

Reply:
maybe rename to ..._BLACKLIST_HEIGHT?

Reply:
yes, the height of block, when blacklist begin to work